webapp/src/components/conference/conference.tsx

```diff
@@ -53,6 +54,12 @@ export default class Conference extends React.PureComponent<Props, State> {
         return Math.max(document.documentElement.clientHeight || 0, window?.innerHeight || 0) - (BORDER_SIZE * 2);
     }
 
+  escFunction = (event: any) => {
+      if (event.keyCode === 27) {
```

Comment:
add a comment that '27' == escape key code?

Reply:
will update it. Thanks